UnitTests/BraintreeVisaCheckoutTests/BTConfiguration+VisaCheckout_Tests.swift

```diff
@@ -0,0 +1,112 @@
+import XCTest
+import VisaCheckoutSDK
+import BraintreeTestShared
+@testable import BraintreeCore
+@testable import BraintreeVisaCheckout
+
+class BTConfiguration_VisaCheckoutTests: XCTestCase {
+    
+    func testIsVisaCheckoutEnabled_whenVisaCheckoutEmptyJSONExists_returnsFalse() {
+        let configurationJSON = BTJSON(value: [
+            "visaCheckout": [:]
+        ])
+        let configuration = BTConfiguration(json: configurationJSON)
+        
+        XCTAssertFalse(configuration.isVisaCheckoutEnabled)
+    }
+    
+    func testIsVisaCheckoutEnabled_whenVisaCheckoutFromConfigurationJSONExists_returnsTrue() {
+        let configurationJSON = BTJSON(value: [
+            "visaCheckout": [
+                "apiKey": "apikey" // key must be 'apiKey' not 'apikey' to match Swift logic
```

Comment:
I'm not sure I follow this comment. Can you provide more details?

Reply:
ahh this comment was copy/pasted from the original obj-c code, removed it as it is no longer needed